src/pattern-library/components/patterns/navigation/LinkButtonPage.tsx

```diff
@@ -106,13 +106,9 @@ export default function LinkButtonPage() {
                   Underline: none (default)
                 </LinkButton>
 
-                <LinkButton href="https://www.example.com" underline="hover">
-                  Underline: hover
-                </LinkButton>
+                <LinkButton underline="hover">Underline: hover</LinkButton>
 
-                <LinkButton href="https://www.example.com" underline="always">
-                  Underline: always
-                </LinkButton>
+                <LinkButton underline="always">Underline: always</LinkButton>
```

Comment:
The `href` attribute was not used in `LinkButton`, so it's safe to remove.

These were probably some old examples from a time in which it may have supported it.